packages/react/src/analytics/integrations/GA4/commands.js

```diff
@@ -12,12 +12,18 @@ import ga4EventNameMapping, {
   getEventProperties,
   InternalEventTypes,
 } from './eventMapping';
+import get from 'lodash/get';
 import isFinite from 'lodash/isFinite';
 
 const genericCommandsBuilder = data => {
   const eventName = ga4EventNameMapping[data.event];
+  const eventProperties = getEventProperties(data.event, data);
+  const eventData = {
+    uuid: get(data, 'context.event.uuid'),
```

Comment:
I think the `uuid` should be after the spread of the `eventProperties` here to avoid the case where the event properties contain a `uuid` parameter as well. Question: Should we have a different name for the `uuid` parameter since it is an injected parameter of the event? Something like `__uuid` to make it explicit that it is being injected by analytics. What do you think?